examples/chat/proto/chat.proto

```diff
@@ -66,3 +68,10 @@ service Chat {
   // Send a message to a chat room.
   rpc SendMessage(SendMessageRequest) returns (google.protobuf.Empty);
 }
+
+message Command {
+  oneof command {
```

Comment:
I took a stab at it but it doesn't seem to generate nicer code, it's just that the enum name changes `Command` to `Value`. I'm learning towards keeping the name `Command` just so that it's clear they belong together, `Value` seems a bit too abstract.